app/controllers/api/event_logs_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Api::EventLogsController < ApplicationController
+  # params:
+  # {
+  #   "page": "1", // default 1
+  #   "perPage": "10", // default 10
+  # }
+  #
+  # GET /api/event_logs/:address
+  def show
+    address = params[:address]
+    event_logs = EventLog.where("address ILIKE ?", address).order(id: :desc).page(params[:page]).per(params[:perPage])
```

Comment:
So what if the caller passes in a `perPage` as `100_000` or even bigger? Will this be guarded?